apps/web/src/components/Community/Settings/Danger/Delete.tsx

```diff
@@ -0,0 +1,100 @@
+import CommunityProfile from '@components/Shared/CommunityProfile';
+import { ExclamationIcon, TrashIcon } from '@heroicons/react/outline';
+import { Errors } from '@lenster/data/errors';
+import type { Community } from '@lenster/types/communities';
+import { Button, Card, Modal, Spinner, WarningMessage } from '@lenster/ui';
+import { t, Trans } from '@lingui/macro';
+import type { FC } from 'react';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+
+interface DeleteProps {
+  community: Community;
+}
+
+const DeleteSettings: FC<DeleteProps> = ({ community }) => {
+  const [showWarningModal, setShowWarningModal] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleDelete = async () => {
+    try {
+      setIsLoading(true);
+    } catch (error) {
+      return toast.error(Errors.SomethingWentWrong);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <Card className="space-y-5 p-5">
+      <CommunityProfile community={community as Community} />
+      <div className="text-lg font-bold text-red-500">
+        <Trans>This will delete your Lens Community</Trans>
+      </div>
+      <p>
+        <Trans>
+          This will permanently delete your Community on the Lenster. Your
+          community members will not be able to access this community. All your
+          community data will be wiped out immediately and you won't be able to
+          get it back.
```

Comment:
```suggestion
          This will permanently delete your community. Your
          community members will not be able to access this community. All your
          community data will be wiped out immediately and you won't be able to
          get it back.
```